MovieHub/MovieHub/Utils/ThemeType.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ThemeType.swift
+//  MovieHub
+//
+//  Created by Duy Nguyễn on 23/04/2024.
+//
+
+import Foundation
+
+enum ThemeType: Int {
+    case unspecified = 0
+    case light = 1
+    case dark = 2
```

Comment:
mặc định nó đã là 0 1 2 rồi nên k cần phải thêm e
```suggestion
    case unspecified
    case light
    case dark
```